client/dashboard/utils/wp-version.ts

```diff
@@ -0,0 +1,26 @@
+import { __ } from '@wordpress/i18n';
+import type { Site } from '../data/types';
+
+export function getWordPressVersionTagName( versionTag: string ) {
```

Comment:
Does this function bring any value, it feels we can just use the strings inline where it's currently used.

Reply:
Not really 100% sure either to be honest, the only difference is that it's translated 🤔